scripts/setup-env-macos.sh

```diff
@@ -1,31 +1,23 @@
 #!/bin/bash
 set -x
 
-# Quick install script for mac
-
-# Install homebrew
+# Install Homebrew.
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 
-# Brew install all packages
+# Brew install all packages.
 brew upgrade
 
 brew install \
-	go \
-	libomp \
-	node \
-	git \
-	pyenv \
-	yarn
+  go \
+  libomp \
+  node \
+  git \
+  pyenv \
+  yarn
 
 brew cask install \
-	docker
+  docker
 
-# Install Python 3.6 and virtualenv
+# Install Python 3.6 and virtualenv.
 pyenv install 3.6.9
 python3 -m pip install virtualenv
```

Comment:
we shouldn't need virtualenv now right? We should probably do `python3.6 -m venv ~/.virtualenvs/determined` so it's similar to the other scripts

Reply:
Makes sense; could you do a quick check that that works? Python is sometimes packaged so that `python -m venv` doesn't work (something about it requiring Pip to be bundled and package managers not wanting to distribute other package managers). I just tested it with `pyenv` on Linux and it looks good, but I wouldn't want to jump to any conclusions about the behavior on macOS (though it's probably also fine).

Reply:
It worked on my Mac 👍